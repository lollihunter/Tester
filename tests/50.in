500 500 500
1 1
5 13
5 171
287 488
182 430
265 397
263 216
9 105
313 169
337 233
165 94
51 73
364 378
412 271
249 332
289 466
134 55
490 187
419 170
486 241
427 455
423 445
337 190
238 137
85 451
180 410
309 28
258 57
155 273
211 311
260 319
368 413
433 460
459 427
110 49
333 329
361 78
483 44
466 458
216 260
345 280
470 490
24 97
458 254
100 421
97 35
203 170
193 413
372 458
430 241
102 294
427 239
50 50
149 478
29 169
315 22
319 391
298 215
227 458
428 427
351 372
472 176
166 181
498 243
101 388
478 409
484 431
266 352
244 261
357 451
482 302
266 192
229 450
450 415
368 314
223 24
256 178
315 354
354 468
141 427
366 308
163 67
115 383
359 220
389 149
448 364
337 461
210 322
357 479
73 392
130 433
80 422
474 485
34 365
210 304
163 479
497 416
489 331
420 155
421 265
34 299
468 179
63 42
66 91
125 3
40 57
91 52
423 237
483 243
133 249
135 92
147 218
431 253
500 65
141 321
102 446
139 106
283 389
248 142
59 332
214 438
77 205
19 122
152 236
45 473
469 183
356 237
110 82
400 58
117 281
113 64
187 273
324 420
220 162
15 228
29 312
204 135
386 318
470 75
130 356
334 407
344 169
394 1
16 263
104 49
367 35
354 100
43 78
65 163
227 459
204 348
208 108
483 51
266 451
357 44
178 168
164 325
380 56
124 206
168 202
339 24
245 456
352 170
323 306
30 24
308 70
472 271
471 87
151 399
248 115
328 39
75 269
436 274
423 481
138 175
119 35
457 336
141 210
175 49
479 121
362 71
172 220
430 196
331 426
224 216
370 384
429 442
294 394
78 279
347 358
261 353
361 21
446 21
310 126
152 337
231 282
163 304
305 442
262 343
459 88
389 153
320 185
67 413
207 124
102 488
164 284
396 98
33 404
289 162
141 84
347 118
34 281
211 114
433 382
49 76
360 246
268 441
260 190
91 478
481 54
368 161
340 152
496 372
416 457
464 365
152 61
399 229
260 212
36 90
211 18
46 108
18 255
259 487
476 174
299 31
233 317
144 134
48 320
396 415
437 384
121 492
341 147
456 381
27 188
372 468
139 184
82 162
284 312
472 220
62 211
303 479
423 457
42 382
277 282
88 369
114 72
78 148
223 280
118 410
197 339
172 96
101 498
204 442
208 389
309 288
216 480
66 159
171 357
342 174
210 160
249 32
245 114
357 88
181 182
438 49
459 277
412 193
361 482
297 208
247 385
103 122
293 61
148 309
96 81
339 98
7 444
350 323
470 397
263 190
114 426
356 469
392 169
132 493
4 362
356 193
264 4
406 340
241 50
133 255
430 118
139 438
240 145
196 455
210 482
100 256
355 357
27 294
461 36
276 214
68 489
269 440
321 459
418 432
417 267
6 297
191 164
60 333
101 450
170 378
57 476
124 123
145 221
455 144
34 221
340 217
483 286
165 226
391 398
447 71
475 156
303 213
417 386
343 232
154 45
224 154
183 176
291 251
45 76
277 240
402 365
40 153
192 51
263 345
485 492
169 112
241 57
83 425
232 172
136 367
360 385
115 171
67 261
125 315
69 249
312 481
346 108
343 280
431 195
295 355
428 473
200 473
57 466
10 208
86 192
443 42
289 177
490 135
354 480
253 493
243 223
396 459
218 349
43 416
201 345
374 363
20 423
60 128
301 295
457 382
29 363
136 481
227 227
226 144
343 385
453 187
370 28
94 223
181 240
114 102
180 259
484 218
24 39
40 470
396 345
271 500
401 419
223 487
497 186
2 241
128 207
156 154
54 98
337 435
340 30
286 457
342 36
271 211
13 188
226 119
287 70
252 189
500 192
68 369
231 298
442 371
86 470
373 458
256 91
354 142
247 286
281 239
117 312
465 178
423 459
8 275
40 451
461 369
240 377
219 27
458 183
475 172
288 223
29 342
94 486
356 349
297 473
180 98
95 107
274 3
225 419
367 361
189 416
141 365
121 287
448 9
56 296
480 466
208 286
116 379
134 471
480 110
487 315
307 447
327 225
281 75
312 365
331 78
440 162
233 36
464 373
250 6
347 115
138 229
99 98
55 69
63 414
484 195
224 159
418 210
277 234
270 349
476 20
256 414
15 83
96 270
21 184
39 452
66 127
324 51
100 287
412 420
465 246
243 307
410 166
141 383
474 53
82 309
84 42
398 65
113 344
63 419
248 260
257 385
381 324
411 206
352 99
256 56
376 249
154 155
500 500
