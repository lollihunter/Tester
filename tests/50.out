68
1 2 165 392 106 53 103 488 107 145 131 256 128 464 389 321 237 401 499 175 246 159 400 178 112 322 463 124 327 388 390 132 448 258 196 414 483 236 15 31 248 378 265 164 36 287 81 75 494 394 340 332 214 415 86 427 460 243 379 288 66 97 67 39 447 93 344 500 
